src/popup/searchModule.js

```diff
@@ -62,6 +64,10 @@ function showNoResultFoundMessage() {
     paragraph.textContent = 'No Images Found. Please enter a different query.';
 
     removeChildNodes(sectionContentPrimary.querySelector('.row'));
+    // This little code fixes the Issue #100, I am adding a new class
```

Comment:
Can you please don't write the comment in first person. Just state what the code is doing (like the comment you wrote below).